### ISSUE TITLE
Can't get the temperature high enough (beyond 37 C)

### ISSUE THREAD

**Issue by Ramblurr** [quantenschaum/egg-incubator#3]
I realize this answer depends on a multitude of variables, but I pose it here nonetheless.

I'm unable to heat up the inside of my insulated box (one simliar to  @quantenschaum's ) beyond ~37 C. My intended application is yogurt incubation so I need internal temperatures between 43 and 46 C. This is driving me a little nuts, because it should work.

My power supply is 12v/1.5A and I'm using 22 AWG stranded hookup wire which can handle the 1.5A.

I'm using 28 gauge A1 Kanthal wire which is rated at 17.7 Ω/meter.

The 0.82 meter of Kanthal wire I'm using is approximately a  15 Ω resistor. Ohm's law tells us that 12 V and 15 Ω = 0.8 Amps. And 12 V with 0.8 Amps = ~10 Watts.

Shouldn't 10 Watts be enough to heat the interior of a 17cmx30cmx35cm insulated box?

Notably, I'm not using the Arduino yet. I'm just testing the heating circuit with the 120mm fan (running at 5V), and the heating element. No MOSFET, arduino, or temperature sensor yet.

In my tests it takes about 2 hours to heat the interior from ~23 C to ~37C where it plateaus.

**Comment by lefterisgar** [comment]
Maybe use a 220VAC bulb in combination with kanthal wire? The bulb would light up to get your machine up to temp and then to save power the Arduino would power it off and use the kanthal wire instead unless the temperature drops.

**Comment by quantenschaum** [comment]
I can highly advise against the use of a mains powered light bulb! 

1. :skull_and_crossbones:  It is dangerous. (risk of electrical shock and fire)
2. :fire:  It gets way too hot and is totally overpowered. It will have a short duty cycle
3. :dash: It is likely too burn out quickly.
4. :baby_chick: It may cause stress to the animals when light is constantly turned on and off. (You may wrap in tin foil, but then you are back at point 1.) 
5. :loudspeaker: When used with a mechanical relay it causes heavy wear on it and the noise is annoying.

I suspect your heating element has too little power. Shorten the wire to increase the power.

It depends on the insulation of the box. You should aim for ~80% duty cycle of the heater when maintaining constant temperature at the desired setpoint. So there is some headroom for the case that the surroundings get cooler and more heating power is required to maintain temperature. But the heating power should be as small as possible to prevent the chicks from getting fried or even fire in case of malfunction. There is no need to reach the setpoint very quickly, gentle changes are better and more natural.

**Comment by lefterisgar** [comment]
@quantenschaum I get your points here and you are correct... however I didn't make clear the fact that I'm talking about a bigger space (compared to a foam box) where even a 120W 12V heating element won't do the trick.....
[EDIT]: However, by reading this I might ditch the bulb solution and invest in creating a diy resistive heating element around the walls for the new season. What do you think about this one? Is a resistive element all around the incubator a good idea? And how many fans? (again talking about a big room like 1 cubic meter + something)
Thanks!

**Comment by quantenschaum** [comment]
I don't know. Make some experiments. What kind of heating does not really matter, it just needs to have the right amount of power, enough, but not too much. Maybe 2 elements of different power, use both to heat up, an then only one of them to maintain temperature. Maybe these heating mats used in 3D printing are a good choice? I think you do not need may fans, just one to distribute the heat equally in the box.

**Comment by lefterisgar** [comment]
@quantenschaum Yes 3D printing mats sound like a very good choice indeed! Thanks so much for all of your suggestions! I'm going be considering every single one of them for the next incubation period so I'll begin to prepare by now! Again thanks so much! Take care!
